webapp/channels/src/components/integrations/installed_incoming_webhook.tsx

```diff
@@ -26,7 +26,10 @@ export function matchesFilter(incomingWebhook: IncomingWebhook, channel: Channel
     }
 
     if (incomingWebhook.channel_id) {
-        if (channel && channel.name.toLowerCase().indexOf(filter) !== -1) {
+        if (channel &&
```

Comment:
combine the two if conditions and use includes instead of indexOf for better clarity on the intentionality and you can also convert the filter to lowercase once before the condition to avoid redundant .toLowerCase() calls.
below is a suggestion of how it might be (haven't validate it 100% works)
```suggestion
if (incomingWebhook.channel_id && channel) {
    const filterLower = filter.toLowerCase();
    const channelMatches = channel.name.toLowerCase().includes(filterLower) || channel.display_name.toLowerCase().includes(filterLower);

    if (channelMatches) {
        return true;
    }
}
```